docs/source/models/intervention_models/intrapartum/acs_intervention.rst

```diff
@@ -0,0 +1,293 @@
+available.
+
+When we fill in the location-specific values for delivery facility rates, ACS coverage, relative risk of mortality with ACS access, 
+and mortality probability (which is also age-specific), this becomes a system of two linear equations with two unknowns (:math:`p(\text{preterm with RDS}|\text{ACS})` 
+and :math:`p(\text{preterm with RDS}|\text{no ACS})`), which we can solve analytically using the same approach as in the :ref:`cpap calibration <cpap_calibration>`. 
+
+**Alternative PAF Derivation**: An alternative, and possibly simpler derivation of the PAF that will calibrate this model comes from the observation that 
+:math:`\text{PAF} = 1 - \frac{1}{\mathbb{E}(\text{RR})}`.  If we define 
+
+.. math::
+
+   p(\text{no ACS}) = \sum_{\text{paths without ACS}} p(\text{path}),
+
+then can use this to expand the identity
+
+.. math::
+
+   \text{PAF}_\text{no ACS} = 1 - \frac{1}{\mathbb{E}(\text{RR})}.
+
+Since our risk exposure has two categories,
+
+.. math::
+
+   \mathbb{E}(\text{RR}) = p(\text{no ACS}) \cdot \text{RR}_\text{no ACS} + (1 - p(\text{no ACS})) \cdot 1.
+
+.. note:: 
+
+  Simulants should have the same propensity value for CPAP (see the :ref:`CPAP intervention model page <intervention_neonatal_cpap>`) and ACS, such that the same
+  simulants are exposed to both interventions (i.e., if coverage of both CPAP and ACS is 50%, then the same 50% of simulants will be getting each intevention).
+
+Assumptions and Limitations
+---------------------------
+
+- We assume that ACS availability captures actual use, and not simply the treatment being in the facility. 
+- We assume that the delivery facility is also the facility where a mother or birthing person will seek care for their preterm infant with RDS.
+- We assume that the relative risk of preterm with RDS mortality with ACS in practice is a value that we can find in the literature. Note: 
+  the value we are using is from [Oladapo-et-al-2020]_, a BMGF-funded multicountry RCT which compared neonatal mortality for women at imminent
+  risk of preterm birth (i.e., expected to give birth in next 48 hours) that received intramuscular dexamethasone (6mg dosage) versus a placebo. 
+  Both groups (test and control groups) in the trial had access to CPAP, so the effect size reported for ACS is specific to ACS in the presence of
+  CPAP, rather than pertaining to the cumulative effect size of ACS and CPAP together. 
+- We assume that ``(RR_ACS | CPAP) = (RR_ACS | no CPAP)``. This is because of the assumption that ACS affects RDS incidence while CPAP affects RDS 
+  case fatality, so by definition CPAP could not modify the effect of RDS under these assumptions it comes too late in the causal chain.
+- We assume that ``(RR_CPAP | ACS) = (RR_CPAP | no ACS)`` despite the fact that ACS acts on outcomes that come earlier in the causal chain than CPAP,
```

Comment:
Let's add this limitation (ACS does not modify the effect of CPAP) to the CPAP intervention page instead of here. Also, out of curiosity I did a quick search and it looks like this study suggests that ACS use was not significantly associated with CPAP success in RDS treatment, which supports our assumption here! Let's add that as supporting evidence in the note of the limitation too :) https://pmc-ncbi-nlm-nih-gov.offcampus.lib.washington.edu/articles/PMC10155133/